migrations/2024-07-30-124102_add_entity_type_to_roles/up.sql

```diff
@@ -0,0 +1,2 @@
+-- Your SQL goes here
+ALTER TABLE roles ADD COLUMN entity_type VARCHAR(64);
```

Comment:
It need not to be nullable, we can put default as merchant.

Reply:
No, we don't have to remove the default at all.

When we insert the roles we are anyways always inserting the entity_type of the role.

Putting it default will remove the burden of running the backfill queries. Like we did for version.